pkg/controllers/report/background/controller.go

```diff
@@ -425,5 +425,12 @@ func (c *controller) reconcile(ctx context.Context, logger logr.Logger, _, names
 		}
 		return err
 	}
+	defer func() {
```

Comment:
Not sure I get the idea, it re-enqueues the report upon each reconciliation? Wouldn't it results in endless loop?

Reply:
It used queue.AddAfter so takes the interval into account.

Reply:
Is this the delay it uses? The report is added after 30s?https://github.com/kyverno/kyverno/blob/f769619eab939faf2936a98a3e2b0d9b837eb653/pkg/controllers/report/background/controller.go#L43

Reply:
Nope it is the configured interval

Reply:
My bad, it should have been `c.forceDelay`, fixing

Reply:
Fixed

Reply:
thanks!